src/nvim/file_search.c

```diff
@@ -1372,6 +1372,11 @@ char *find_file_in_path_option(char *ptr, size_t len, int options, int first, ch
           memmove(ptr, ptr + 1, strlen(ptr));
         }
       }
+      for (ptr = path_option; *ptr != NUL; ptr++) {
+        if (ptr[0] == '\\' && ptr[1] == ' ') {
+          memmove(ptr, ptr + 1, strlen(ptr));
+        }
+      }
```

Comment:
This is changing the value of the option itself. At least it should make a copy first.

Reply:
My bad, I'm not a C developer. Didn't thought about the consequences of changing an in parameter.